### ISSUE TITLE
including WCAG supplements and Making Content Usable for People with Cognitive and Learning Disabilities 

### ISSUE THREAD

**Issue by lseeman** [w3c/wcag2ict#77]
Can wcag2ict address including the supplements? Specifically including  Making Content Usable for People with Cognitive and Learning Disabilities 

If it is not included at all these groups will be sidelined ongoing.

**Comment by lseeman** [comment]
please email me when you need feedback and or add the coga tracking tag. I find I can not track github comments threads

**Comment by maryjom** [comment]
@lseeman The scope set in the [WCAG2ICT work statement](https://www.w3.org/WAI/GL/task-forces/wcag2ict/work-statement#scope) does not include analyzing any of the supplemental content outside of the WCAG standard itself. Focus is currently on Level A and AA criteria that are utilized, referenced or included in other standards such as Revised 508 Standards or the EN 301 549, for example.

**Comment by maryjom** [comment]
DRAFT ANSWER:

Now that the WCAG2ICT document is nearer to completion, we found a place where this COGA non-normative resource can be referenced.  The Document Overview section already lists some non-normative reference documents for WCAG, and we think that this is a good place for the content usable document reference can be added. See Pull Request #287 for details.

**Comment by lseeman** [comment]
HI ! Thanks so much for your considerations.
You wrote 

I think Additional supporting documents for WCAG 2, such as the [WCAG 2 Overview](https://www.w3.org/WAI/intro/wcag.php), [Techniques for WCAG 2.2](https://www.w3.org/WAI/WCAG22/Techniques/) [[WCAG22-TECHS](https://deploy-preview-287--wcag2ict.netlify.app/#bib-wcag22-techs)], [How to Meet WCAG (Quick Reference)](https://www.w3.org/WAI/WCAG22/quickref/), and [Making Content Usable for People with Cognitive and Learning Disabilities](https://www.w3.org/TR/coga-usable/), remain available for web content, but have not been changed to apply to non-web documents and software.


My response is that Making Content Usable for People with Cognitive and Learning Disabilities discussed  other forms of software. including speech systems,  bots, apps, media, etc.

**Comment by lseeman** [comment]
I think it is worth saying that it should be reviewed to see where it is appropriate for your software, remembering that there are people with learning and cognitive disabilities in every audience. .

**Comment by maryjom** [comment]
@lseeman 

Further adjustments have been made to the language and location of documents that are referred to by WCAG2ICT, including "Making Content Usable..." that will hopefully address your concerns in the additional comments (above) in this issue. See PR #309 for the details of the changes made. 

Note that the WCAG2ICT Task Force needs to remain mindful that this document is not meant to provide techniques for meeting or going beyond WCAG. Its content is limited to interpreting WCAG criteria in a non-web context. Any guidance beyond that should instead be included in or referenced by other documents specifically focused on implementing accessibility requirements in non-web technologies. We will continue to mention this resource, but in the context we feel is appropriate for our scope of work.

Here is the exact language that the WCAG2ICT Task Force agreed on which can be found in the section titled [Guidance in this Document](https://w3c.github.io/wcag2ict/#guidance-in-this-document) starting the 2nd paragraph after Note 1. 
>Although this document covers a wide range of issues, it is not able to address all the needs of all people with disabilities. Since WCAG 2 was developed for the Web, addressing accessibility for non-web documents and software may involve requirements and considerations beyond those included in this document. Authors and developers are encouraged to seek relevant advice about current best practices to ensure that non-web documents and software are accessible, as much as possible, to people with disabilities. The following WCAG 2 supporting documents, though they have not been changed to fully apply in non-web contexts, contain helpful information to learn about the user needs, intent, and generalized implementation techniques:
>
>* [WCAG 2 Overview](https://www.w3.org/WAI/intro/wcag.php)
>* [Techniques for WCAG 2.2](https://www.w3.org/WAI/WCAG22/Techniques/) [[WCAG22-TECHS](https://w3c.github.io/wcag2ict/#bib-wcag22-techs)]
> * [How to Meet WCAG (Quick Reference)](https://www.w3.org/WAI/WCAG22/quickref/)
>* [Making Content Usable for People with Cognitive and Learning Disabilities](https://www.w3.org/TR/coga-usable/)
>* [Mobile Accessibility: How WCAG 2.0 and other W3C/WAI Guidelines Apply to Mobile](https://w3c.github.io/Mobile-A11y-TF-Note/)

**Comment by maryjom** [comment]
Closing as answered.

**Comment by lseeman** [comment]
I am having trouble opening a new issue so I am addressing related follow on issues here

The bellow  sentence is misleading .
 _Addressing accessibility involves addressing the needs of people with auditory, cognitive, neurological, physical, speech, and visual impairments, as well as accessibility needs of people due to the effects of aging._

It suggest equity between the different user groups. but the document does not address primary needs for  cognitive,. That is why there is a separate supplement.

the link bellow does not represent that reality either.  reference to  Coga as being in a list of documents that you may find useful that you are under no obligation to open. For example, the quick ref may be useful, but if you are happy with the current document, there is no need to look at it. 

I suggest replacing the sentence as follows 
Addressing accessibility involves addressing the needs of people with auditory, 
, neurological, physical, speech, and visual impairments, as well as accessibility needs of people due to the non-cognitive  effects of aging. Although this document addresses some user needs for people with cognitive and learning disability and mental health related disabilities we recommend also following the WCAG supplement [Making Content Usable for People with Cognitive and Learning Disabilities](https://www.w3.org/TR/coga-usable/) to address user needs of these groups as well as well as some mental health related disabilities .  

In addition I would change the heading from : Guidance in this Document
to Limited Scope of Guidance in this Document

If we are not giving equity in our new publications , the least we can do is be clear about who we are excluding in plain language so that people will understand that other documents must be looked at to include people with disability's equitably.

**Comment by lseeman** [comment]
A preferred approach would be to also add an additional sentence that
 The taskforce  are working on an ict version of the design guided for the supplement, however the same conversion principles can usually be applied. 
(and actually make one)

**Comment by rachaelbradley** [comment]
(Chair hat on) I do not believe it is within the current iteration of the WCAG2ICT taskforce to write an ICT version of Making Content Usable.

**Comment by rachaelbradley** [comment]
(Chair hat off) I agree that the wording in the current document does not meet the original intent of this issue. While I would like to see this document reference a fully published note or notes from the Low Vision, Mobile, and COGA taskforces, only the COGA taskforce has published one. The others are all in some state of draft.  

I also do not believe that Making Content Usable is the same type of document as the WCAG 2 Overview, Techniques, and How to Meet Documents and they do not fit together well in the list.  

I am late to this conversation and will not hold up publication of the FPWD based on this wording but I do not support it going forward to final publication as written.

**Comment by maryjom** [comment]
WCAG2ICT TF answer to the thread of comments after the issue was reopened (Comments starting with [Lisa's comment from 13 June](https://github.com/w3c/wcag2ict/issues/77#issuecomment-2164527902)):

On 14 June the WCAG2ICT Task Force reached a Resolution on the following:

1. We would not update the heading "Guidance in this Document". This heading matches what was in the 2013 WCAG2ICT and does describe the content contained in the section.
2. We accepted, with some editorial changes, what was proposed in Lisa's comment. This changes the first paragraph after the Note 1 to read:
>This document is intended to help clarify how to use WCAG 2 to make non-web documents and software more accessible to people with disabilities. Addressing accessibility involves addressing the needs of people with auditory, cognitive, neurological, physical, speech, and visual impairments, as well as accessibility needs of people due to the effects of aging. Although WCAG 2 addresses some user needs for people with cognitive and learning disabilities as well as mental health related disabilities, following the WCAG supplement [Making Content Usable for People with Cognitive and Learning Disabilities](https://www.w3.org/TR/coga-usable/) is recommended for non-web ICT to address the user needs of these groups.
3. We removed the reference to the _Making Content Usable_ document from the bulleted list of WCAG 2 supporting documents.
4. We did not add the [additional suggested sentence offered](https://github.com/w3c/wcag2ict/issues/77#issuecomment-2164556090). Instead the AG WG plans to create a page where the various Task Forces (COGA, Low Vision, Mobile Accessibility, etc.) can post their draft documents and references that would be helpful (per issue #383) for implementers of WCAG. This particular change to the link in WCAG2ICT will occur once that page of references is ready.

These changes can be found in the file [introduction.md in PR 378](https://github.com/w3c/wcag2ict/pull/378/files#diff-f90b081e513dea80603790c785f691eddd100e03687bf921e64190f99c315d9e). They can be read in-context in the built document for the PR [Guidance in this Document section](https://deploy-preview-378--wcag2ict.netlify.app/#guidance-in-this-document) (first paragraph after Note 1).  The TF intends to merge in this PR once the AG WG approves it on Tuesday 25 June.

**Comment by maryjom** [comment]
Closing this issue as answered. The WCAG2ICT document is now in CfC to publish that ends on 1 July. Then the document will be published for a 30-day wide review prior to being finalized. Any further comments should be provided in a [new issue](https://github.com/w3c/wcag2ict/issues/new).
